logger/src/dal.rs

```diff
@@ -254,6 +256,30 @@ impl Log {
             })
             .collect();
 
+        let mut fields = from_any_value_kv_to_serde_json_map(span.attributes);
+        fields.insert(
+            MESSAGE_KEY.to_string(),
+            format!("[span] {}", span.name).into(),
+        );
+
+        logs.push(Log {
+            shuttle_service_name: shuttle_service_name.to_string(),
+            deployment_id,
+            timestamp: DateTime::from_utc(
+                NaiveDateTime::from_timestamp_opt(
+                    (span.start_time_unix_nano / 1_000_000_000)
+                        .try_into()
+                        .unwrap_or_default(),
+                    (span.start_time_unix_nano % 1_000_000_000) as u32,
+                )
+                .unwrap_or_default(),
+                Utc,
+            ),
+            // Span level doesn't exist so this info is not relevant.
+            level: LogLevel::Info,
```

Comment:
Wait, doesn't `tracing` allow setting the span level?

Reply:
Sadly, I couldn't find anything. Also this: https://github.com/open-telemetry/opentelemetry-specification/issues/3205

Reply:
Oh, interesting... A `tracing` span does have a verbosity level (it's [required](https://docs.rs/tracing/latest/tracing/struct.Metadata.html)), but from your link Iulian it seems it's not included in the exported trace since it's not a part of the otel spec. Perhaps we can work around it somehow by adding the span levels to the span attributes? :thinking: (perhaps this should also be explored in a follow-up PR)

Reply:
Yeh, I will try this tomorrow. 👌